frontend/test/metabase/scenarios/setup/user_settings.cy.spec.js

```diff
@@ -11,12 +11,13 @@ describe("user > settings", () => {
     cy.signInAsNormalUser();
   });
 
-  it("should show user details", () => {
+  it("should show user details with disabled submit button", () => {
     cy.visit("/user/edit_current");
     cy.findByText("Account settings");
     cy.findByDisplayValue(first_name);
     cy.findByDisplayValue(last_name);
     cy.findByDisplayValue(email);
+    cy.findByText("Update").should("be.disabled");
```

Comment:
```suggestion
    cy.findByRole("button", { name: "Update" }).should("be.disabled");
```
`cy.findByText(string)` is most likely not going to work here because it literary finds the string that we use inside the button. And most of our buttons have an actual `button` element with `span` or `div` inside. So, what ends up happening is that Cypress tries to assert that span/div are (not) disabled.

You can read more about it (and see it in action) in this PR: https://github.com/metabase/metabase/pull/14833

Btw, I could've sworn there was already a repro for this, but I can't find it now. 🤔

Reply:
Fixed this, thank you!